ra/next/ra-api.ng

```diff
@@ -24,6 +24,9 @@
 			<optional>
 				<attribute name="required"> <ref name="boolean-values" /> </attribute>
 			</optional>
+			<optional>
+				<attribute name="replaced-by"> <text /> </attribute>
+			</optional>
```

Comment:
To digress for a moment, I'm leaning toward a series of small jumps for a better historical record: 1.1 to codify existing usage as much as possible; 1.2 with backward-compatible new features such as this one; and 2.0 with backward-incompatible changes. Perhaps in "next" requests like this we could tag the changes with the targeted version, e.g. <!-- 1.2 -->

Reply:
okay, I'm fine with waiting for 1.2

This feature is already used in fence agents (both upstream & RHEL7)